flutter/example/lib/main.dart

```diff
@@ -116,6 +116,18 @@ class MainScaffold extends StatelessWidget {
                 Sentry.captureMessage('A message with a print() Breadcrumb');
               },
             ),
+            RaisedButton(
+              child:
+                  const Text('Capture message with scope without breadcrumbs'),
+              onPressed: () {
+                Sentry.captureMessage(
+                  'This event has no breadcrumbs',
+                  withScope: (scope) {
+                    scope.clearBreadcrumbs();
```

Comment:
instead of only removing data which is always nice to see, lets do something else, like, adding a tag, an extra, or something else